data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_active_on_offense.cfg

```diff
@@ -0,0 +1,33 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]active_on=
+##
+# Actions:
+# Give all units resistance to all damage types with a value of 50
```

Comment:
Doesn't match the offensive-only ability

Reply:
Updated.